src/ducks/balance/components/AccountRow.styl

```diff
@@ -96,6 +96,9 @@
     .AccountRow__logo
         opacity 0.5
 
+.AccountRow--withOwners
+    +small-screen()
+        height rem(82)
```

Comment:
What is this magic value ?

Reply:
Fixing https://github.com/cozy/cozy-banks/pull/1696#discussion_r353166734 would also fix this magic value issue. I will try something